src/Altinn.Platform/Altinn.Platform.Storage/IntegrationTest/Altinn.Platform.Storage.IntegrationTest.csproj

```diff
@@ -8,18 +8,33 @@
     <ProjectGuid>{a6ba108e-5155-4aa7-a441-4a1852b46191}</ProjectGuid>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
why are we both remocing and including the same resource?

Reply:
This is something Visual Studio does sometimes when it comes to resource files like this. I'm not sure why. Not fixing.